phpstan.neon.dist

```diff
@@ -0,0 +1,9 @@
+includes:
+    - phpstan-baseline.neon
+
+parameters:
+	level: 0
+	paths:
+		- src
+	excludes_analyse:
+		- src/JiraRestApiServiceProvider.php
```

Comment:
Must be excluded from analyse because of `use Illuminate\Support\ServiceProvider;` which is only present if lavarel framework is used